codecs/avif/Makefile

```diff
@@ -0,0 +1,75 @@
+ROOT_DIR = ./node_modules
+CODEC_DIR_RELATIVE = libavif
+CODEC_DIR = $(addprefix $(ROOT_DIR)/, $(CODEC_DIR_RELATIVE))
+CODEC_OUT_RELATIVE = build/libavif.a
+CODEC_OUT := $(addprefix $(CODEC_DIR)/, $(CODEC_OUT_RELATIVE))
+
+CODEC_VERSION = "v0.8.0"
+
+LIBAOM_RELATIVE = ext/aom/
```

Comment:
Let's avoid extra vars and `addprefix` where they're not necessary.

In this case, I suppose we won't have more than one folder, and `addprefix` is mainly useful for things like targets (where you might have >1 and want to add the same prefix to each of them). Instead, in this case it could be just

```
LIBAOM_DIR := $(CODEC_DIR)/ext/aom
```

Same for some other cases around.

Reply:
> keep it simple

That ship has sailed :P

Okay, I’m happy to reduce number of variables.

Reply:
It's not even that much number of variables that bothers me, as complexity of them layered with `addprefix` combinations to figure out the final value. I think simple concat should be enough for anything except target list :)